tests/integration/size/test_strip_symbols_insight.py

```diff
@@ -0,0 +1,42 @@
+import platform
+
+from pathlib import Path
+
+import pytest
+
+from launchpad.artifacts.apple.zipped_xcarchive import ZippedXCArchive
+from launchpad.size.analyzers.apple import AppleAppAnalyzer
+
+
+class TestStripSymbolsInsightIntegration:
+    def test_strip_symbols_with_hackernews_analyzer(self):
+        # Skip this test if we are not on a mac
+        if not platform.system() == "Darwin":
```

Comment:
Test is skipped on non-mac systems since it doesn't work.

Should I add a mac agent to run this test?

Reply:
The `T` option for the `strip` command is invalid.
https://github.com/getsentry/launchpad/actions/runs/16197471524/job/45727625524#step:7:287

From the man page, it seems this is important:
```
 The intent of this flag is to remove Swift symbols from the Mach-O symbol table, It removes the symbols whose names begin with `_$S' or `_$s' only when it finds an __objc_imageinfo section with and
              it has a non-zero swift version.  The future the implementation of this flag may change to match the intent.  When used together with -R,/ -s files the Swift symbols will also be removed from global
              symbol lists used by dyld.
```